src/patternfly/components/Alert/styles.scss

```diff
@@ -1,80 +1,68 @@
-  --pf-c-alert__link--FontWeight: var(--global--FontWeight--bold);
-  //success alert
-  --pf-c-alert--is-success--BackgroundColor: var(--global--success-color);
-  --pf-c-alert--is-success--Color: var(--global--success-color--dark);
-  --pf-c-alert--is-success--BorderColor: var(--global--success-color--dark);
-  //danger alert
-  --pf-c-alert--is-danger--BackgroundColor: var(--global--danger-color);
-  --pf-c-alert--is-danger--Color: var(--global--Color--light);
-  --pf-c-alert--is-danger--BorderColor: var(--global--danger-color--dark);
-  //warning alert
-  --pf-c-alert--is-warning--BackgroundColor: var(--global--warning-color);
-  --pf-c-alert--is-warning--Color: var(--global--warning-color--dark);
-  --pf-c-alert--is-warning--BorderColor: var(--global--warning-color--dark);
-  //info alert
-  --pf-c-alert--is-info--BackgroundColor: var(--global--info-color);
-  --pf-c-alert--is-info--Color: var(--global--info-color--dark);
-  --pf-c-alert--is-info--BorderColor: var(--global--info-color--dark);
-}
-//
-// Base styles
-//
+
+$pf-c-alert--PaddingTop:        $global--spacer--sm !default;
+$pf-c-alert--PaddingRight:      $global--spacer--md !default;
+$pf-c-alert--PaddingBottom:     $global--spacer--sm !default;
+$pf-c-alert--PaddingLeft:       $global--spacer--md !default;
+$pf-c-alert--MarginBottom:      $global--spacer !default;
+$pf-c-alert--BorderWidth:       $global--BorderWidth !default;
+$pf-c-alert__link--FontWeight:  $global--FontWeight--bold !default;
+
+//success alert
+$pf-c-alert--is-success--BackgroundColor:  $global--success-color !default;
+$pf-c-alert--is-success--Color:            $global--success-color--dark !default;
+$pf-c-alert--is-success--BorderColor:      $global--success-color--dark !default;
+
+//danger alert
+$pf-c-alert--is-danger--BackgroundColor:  $global--danger-color !default;
+$pf-c-alert--is-danger--Color:            $global--Color--light !default;
+$pf-c-alert--is-danger--BorderColor:      $global--danger-color--dark !default;
+
+//warning alert
+$pf-c-alert--is-warning--BackgroundColor:  $global--warning-color !default;
+$pf-c-alert--is-warning--Color:            $global--warning-color--dark !default;
+$pf-c-alert--is-warning--BorderColor:      $global--warning-color--dark !default;
+
+//info alert
+$pf-c-alert--is-info--BackgroundColor:  $global--info-color !default;
+$pf-c-alert--is-info--Color:            $global--info-color--dark !default;
+$pf-c-alert--is-info--BorderColor:      $global--info-color--dark !default;
 
 .pf-c-alert {
   position: relative;
-  padding: var(--pf-c-alert--PaddingTop, .5rem) var(--pf-c-alert--PaddingRight, 2rem) var(--pf-c-alert--PaddingBottom, .5rem) var(--pf-c-alert--PaddingLeft, 2rem);
-  background-color: var(--pf-c-alert--BackgroundColor, transparent);
-  border: var(--pf-c-alert--BorderWidth, 1px) solid transparent;
+  padding: $pf-c-alert--PaddingTop, $pf-c-alert--PaddingRight, $pf-c-alert--PaddingBottom, $pf-c-alert--PaddingLeft;
```

Comment:
Looks like the commas didn't get stripped out. I bet they are left over from the default values from the custom properties.